runtime/doc/options.txt

```diff
@@ -5238,7 +5238,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	  print/	files for printing |postscript-print-encoding|
 	  spell/	spell checking files |spell|
 	  syntax/	syntax files |mysyntaxfile|
-	  tutor/	files for vimtutor |tutor|
+	  tutorials/	tutorial files |:Tutor|
```

Comment:
@fmoralesc If you can change this directory name, I'm good with merging this. Don't worry about the syntax comment below.